scripts/convertMdToHtml.js

```diff
@@ -103,6 +103,6 @@
   }
 
   function addForkMeRibbon(fileName) {
-    return '<a href="https://github.com/bonitasoft/bonita-doc/edit/'+version+'/md/'+fileName+'"><img style="position: absolute; top: -1px; right: 0; border: 0;" src="https://camo.githubusercontent.com/365986a132ccd6a44c23a9169022c0b5c890c387/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f7265645f6161303030302e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_red_aa0000.png"></a>\n'
+    return '<div style="position: relative;"><a href="https://github.com/bonitasoft/bonita-doc/edit/'+version+'/md/'+fileName+'"><img style="position: absolute; top: -1px; right: -40px; border: 0; z-index=-100;" src="https://camo.githubusercontent.com/365986a132ccd6a44c23a9169022c0b5c890c387/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f7265645f6161303030302e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_red_aa0000.png"></a></div>\n'
```

Comment:
Maybe can change the z-index? to avoid table to override ribbon,

![image](https://user-images.githubusercontent.com/4995528/29510569-8be3d26a-865c-11e7-8895-6e7f0faa23ac.png)

But it is a very small issue, that appear in very specific case.
Else that works fine, even using Edge or IE

Reply:
What is better? To hover the text or to partially hide the Github Ribbon? I don't know...

Reply:
By the way, your screenshot is WITHOUT the fix, as we can see the scrollbar issue!